components/domobject_derive/Cargo.toml

```diff
@@ -10,5 +10,5 @@ path = "lib.rs"
 proc-macro = true
 
 [dependencies]
-syn = "0.11"
-quote = "0.3.15"
+syn = "0.12"
+quote = "0.4"
```

Comment:
You sure about these versions, and that you don't require anything introduced in a minor release of either? Note that Gecko will use *exactly* 0.12 and 0.4 and thus it could fail to compile there. Just to be safe I would recommend setting the lowest bounds to 0.12.12 and 0.4.2.

Reply:
👍